boa/src/builtins/global_this/tests.rs

```diff
@@ -0,0 +1,10 @@
+use crate::exec;
+
+#[test]
+fn global_this_exists_on_global_object_and_evaluates_to_global_this_value() {
+    let scenario = r#"
+        globalThis;
```

Comment:
The test is failing because the `globalThis` is the global object, it's not the string value `"globalThis"`, when we call `global_this::init` we set the field "globalThis" (this is the binding name not the value) and the value is the global object (that holds objects like `Number`, `String`, etc), a better way of checking if it exists would be to check `globalThis !== undefined` this evaluates to `"true"`

Reply:
Ah, ok! I understand! Thank you for your patience, @HalidOdat! :)